src/component/application/application-list-component.js

```diff
@@ -25,49 +25,46 @@ const Empty = () => (
     </React.Fragment>
 );
 
-class ClientStrategies extends Component {
-    static propTypes = {
-        applications: PropTypes.array,
-        fetchAll: PropTypes.func.isRequired,
-        settings: PropTypes.object.isRequired,
-        updateSetting: PropTypes.func.isRequired,
-    };
+const ClientStrategies = props => {
```

Comment:
Just a small note. We've kept a pattern of destructuring inline: 
```suggestion
const ClientStrategies = ({ fetchAll, applications }) => {
```

Would be good to keep the consistency, and it also makes it easy to see component dependencies at a glance.